src/constans/EndPoints/ENDPOINT.tsx

```diff
@@ -1,18 +1,18 @@
-const BASEURL = `https://upskilling-egypt.com:3005/api`;
-
-// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<AUTHURL>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-const AUTHURL = {
-  REGISTER: `${BASEURL}/auth/register`,
-  LOGIN: `${BASEURL}/auth/login`,
-  FORGET: `${BASEURL}/auth/forgot-password`,
-  RESET: `${BASEURL}/auth/reset-password`,
-  CHANGE: `${BASEURL}/auth/change-password`,
-  LOGOUT: `${BASEURL}/auth/logout`,
-};
-// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-
-// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<INSTRUCTORSURL>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-
-const DASHINSTRUCTORS = {
-  GETFRISTFIVE: `${BASEURL}/instructors`,
-};
+const BASEURL = `https://upskilling-egypt.com:3005/api`;
```

Comment:
I suggest search for axios instances and axios interceptors. They will help us organize the api logic better and handle getting the token before making a request which will eliminate the need to access the token in each network request.